src/renderer/lib/playlist.js

```diff
@@ -38,7 +38,7 @@ function getPreviousIndex (state) {
 
 function getCurrentLocalURL (state) {
   return state.server
-    ? state.server.localURL + '/' + state.playing.fileIndex
+    ? state.server.localURL + '/' + state.playing.fileIndex + '/' + state.window.title
```

Comment:
Getting the file name off of the window title is not ideal. What if we change the Window title to not be an exact match of the filename in the future?

We should add the file name to `state.playing` and then reference it here.

Reply:
> We should add the file name to state.playing and then reference it here.

That's right! 😉

> Getting the file name off of the window title is not ideal. What if we change the window title to not be an exact match of the filename in the future?

I don't have any better idea at the moment, so could we leave this for the future?